spotify-client.spec

```diff
@@ -183,6 +182,15 @@ fi
 #%{_datadir}/applications/spotify.desktop
 
 %changelog
+* Fri Apr 17 2015 Damien Radtke <damienradtke@gmail.com>
+- fetch SUSE and Spotify versions on the fly
+
+* Fri Mar 21 2014 Stefan Botter <obs@botter.cc>
```

Comment:
To nitpick, technically adding these older changes to `%changelog` should be done in a separate first commit, since IIUC they're not related to your changes.  But I'll turn a blind eye if you can't be bothered to fix this ;-)